osu.Game/Graphics/UserInterface/ScreenTitle.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class ScreenTitle : CompositeDrawable, IHasAccentColour
+    {
+        private readonly SpriteIcon iconSprite;
+        private readonly OsuSpriteText titleText, pageText;
+
+        public FontAwesome Icon
+        {
+            get => iconSprite.Icon;
+            set => iconSprite.Icon = value;
+        }
+
+        public string Title
+        {
+            get => titleText.Text;
+            set => titleText.Text = value;
+        }
+
+        public string Page
+        {
+            get => pageText.Text;
+            set => pageText.Text = value;
+        }
+
+        public Color4 AccentColour
+        {
+            get => pageText.Colour;
+            set => pageText.Colour = value;
+        }
+
+        public ScreenTitle()
+        {
+            AutoSizeAxes = Axes.Both;
+
+            InternalChildren = new Drawable[]
+            {
+                iconSprite = new SpriteIcon
+                {
+                    Size = new Vector2(25),
+                    Anchor = Anchor.TopLeft,
+                    Origin = Anchor.TopRight,
```

Comment:
This puts the icon outside the left bounds of the composite, which is a little unsafe in the case of masking. Should be `TopLeft`.

Reply:
I deliberately changed it to this so that it would be easier to align it without using "magic numbers". Do you have any better suggestions than hardcoding the offset like before?

Reply:
As per the current osu-web designs, which will be used going forward for the client, the title is icon-aligned to the content. So I think if any current designs require an offset to get the text aligned instead then they should be responsible for that themselves.

Reply:
Can you reply with an image of that design for the multiplayer screen? Might as well implement it if I'm at it.

Reply:
We're probably doing the new design application in a single swoop, so I'd kinda advise against that for now (just aiming to structure the logic as close to the new designs as we can to reduce the overhead when implementing the next version).